frame/support/test/tests/construct_runtime.rs

```diff
@@ -137,3 +148,11 @@ fn check_module2_error_type() {
 		Err(DispatchError::Module { index: 2, error: 0, message: Some("Something") }),
 	);
 }
+
+#[test]
+fn integrity_test_works() {
+	// If __construct_runtime_integrity_test::test is executed then the atomic value will be
+	// modified two times in the test suite.
+	__construct_runtime_integrity_test::test();
+	assert!(INTEGRITY_TEST_EXEC.load(Ordering::Relaxed) > 1);
```

Comment:
cargo test seems to always execute `__construct_runtime_integrity_test::test` before, we could have assert value is 2 but I prefer not to rely on execution order of tests.

Reply:
stimmt 👍